modules/wiki/set.py

```
@@ -42,10 +40,9 @@ async def _(msg: Bot.MessageSession, interwiki: str, wikiurl: str):
     target = WikiTargetInfo(msg)
     check = await WikiLib(wikiurl, headers=target.get_headers()).check_wiki_available()
     if check.available:
-        if msg.target.target_from in audit_available_list:
+        if not Url.disable_mm:
             if check.value.in_blocklist and not check.value.in_allowlist:
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

(SIM102)